utils/blech_process_utils.py

```diff
@@ -58,27 +77,66 @@ def return_training_set(self, data):
                              int(np.min((data.shape[0], self.dat_thresh))))]
         return train_set
 
-    def fit_model(self, train_set, clusters):
+    def fit_manual_model(self, train_set, clusters):
         """
         Cluster waveforms
         """
         model = gmm(
             n_components=clusters,
-            max_iter=self.params_dict['num_iter'],
-            n_init=self.params_dict['num_restarts'],
-            tol=self.params_dict['thresh']).fit(train_set)
+            max_iter=self.params_dict['clustering_params']['num_iter'],
+            n_init=self.params_dict['clustering_params']['num_restarts'],
+            tol=self.params_dict['clustering_params']['thresh'],
+            ).fit(train_set)
         return model
 
+    def fit_auto_model(self, train_set, clusters):
+        """
+        Since the dirichlet process WILL find more clusters for more data,
+        (whereas we know that more spikes doens't mean more neurons),
+        we need to keep the "COUNT" constant
+        One way to do this is to use KMeans centroids as data points
+        """
+
+        max_k = 1000
```

Comment:
again, recommend params

Reply:
Again, I don't think this value will change unless the number of spikes we normally observe increase by orders of magnitude. I think of it as less of a parameter and more of a constant.
What say you?